modules/complete-onpremise-cluster/README.md

```diff
@@ -0,0 +1,16 @@
+```hct
```

Comment:
This module is already in onpremise repo, so does not make sense to name it -onpremise, just use `complete-cluster`.

Reply:
esmeky miqich lav chkaskaca

Reply:
Just rename the module to `complete-cluster`.